### PR TITLE
clang-8.0: update to 8.0.1 patch release

### PR DESCRIPTION
**Pull request by cjones051073**
Update to 8.0.1 patch release.

Tested on macOS10.14 by compiling a number of ports using it.

@kencu FYI

### CODE REVIEW

**Comment by chrstphrchvz** [comment]
`py-clang` should probably also be updated (maybe the `llvm-*` ports should have a comment reminding to do this).

File: lang/llvm-8.0/Portfile [hunk 1]
```diff
@@ -121,34 +124,34 @@ if {${distfiles} ne ""} {
...
+                    sha256  187179b617e4f07bb605cc215da0527e64990b4a7dd5cbcc452a16b64e02c3e1 \
+                    size    1994068 \
+                    lldb-${version}.src.tar.xz \
+                    rmd160  063b9faece0d5f4dadb0ed44e499581a631d17d9 \
+                    sha256  e8a79baa6d11dd0650ab4a1b479f699dfad82af627cbbcd49fa6f2dc14e131d7 \
+                    size    19586288 \
+                    polly-${version}.src.tar.xz \
+                    rmd160  d753179e2464812fe7da5e62471ad17819936ed5 \
+                    sha256  e8a1f7e8af238b32ce39ab5de1f3317a2e3f7d71a8b1b8bbacbd481ac76fd2d1 \
+                    size    8711692
```
Comment (kencu) [inline lang/llvm-8.0/Portfile]: I'm pretty sure Jeremy just usually deletes all these checksums from the portfile, and the runs "sudo port -v checksum" and copies/pastes the results back into the portfile. Ergo no lldb-${version} stuff.

Comment (cjones051073) [reply lang/llvm-8.0/Portfile]: Yeah, that's almost the way I do it, I just don't copy the whole thing on mass. What I had was exactly equivalent but no problem, I have changed it to be the same procedure.

**Comment by cjones051073** [comment]
> `py-clang` should probably also be updated (maybe the `llvm-*` ports should have a comment reminding to do this).

I would rather leave updating that to someone who knows how to test it.

**Comment by cjones051073** [comment]
One thing I should have noted at first is the master site used for downloading the source tarballs has changed, to match what it looks like upstream is doing as part of their migration from svn to GitHub. i.e. links

https://releases.llvm.org/${version}

was changed to

https://github.com/llvm/llvm-project/releases/download/llvmorg-${version}

to match what is advertised for 8.0.1 at

https://releases.llvm.org/download.html#8.0.1

The old link in fact does not have the tarballs for 8.0.1 (which I believe is probably intentional).

**Comment by cjones051073** [comment]
@eborisch FYI, as py-clang maintainer. Please feel free to push your own commits to this branch to update py-clang, or just open your own PR for it. Note my comment above about the change in master site, as I think you will also need to adapt.

**Pull request merged**